spec/requests/api/alerts_spec.rb

```diff
@@ -0,0 +1,48 @@
+describe "Alerts API" do
+  let(:alert_status)  { FactoryGirl.create(:miq_alert_status) }
+  let(:alert_status2) { FactoryGirl.create(:miq_alert_status) }
+  let(:alert_url) { alerts_url(alert_status.id) }
+  let(:vm) { FactoryGirl.create(:vm) }
+  let(:alert_definition) { FactoryGirl.create(:miq_alert, :severity => "info") }
+  let(:user) { FactoryGirl.create(:user_with_group) }
+
+  it "forbids access to actions without an appropriate role" do
+    api_basic_authorize
+    alert_status
+    alert_status2
+    run_get(alerts_url)
+
+    expect(response).to have_http_status(:forbidden)
+  end
+
+  it "reads all alerts statuses" do
+    api_basic_authorize collection_action_identifier(:alerts, :read, :get)
+    alert_status
+    alert_status2
+    run_get(alerts_url)
+    expect(response).to have_http_status(:ok)
+
+    expect(response.parsed_body["count"]).to eq(2)
+  end
+
+  # TODO: This test can be used after https://github.com/ManageIQ/manageiq/pull/13233 merges
+  # context "alerts subcollection" do
+  #   let(:infra_provider) { FactoryGirl.create(:ems_infra) }
+  #   let(:vm) { FactoryGirl.create(:vm) }
+  #   before(:each) {
+  #     api_basic_authorize collection_action_identifier(:providers, :read, :get)
+  #   }
+  #
+  #   it "get all alertss for a provider" do
+  #     alert_status2
+  #     alert_status
+  #     infra_provider
+  #     infra_provider.vms << vm
+  #     vm.miq_alert_statuses << alert_status2
+  #     run_get("#{providers_url}?expand=resources,alerts")
+  #     expect(response).to have_http_status(:ok)
+  #     expect(response.parsed_body["resources"].first["alerts"]["id"]).to eq(alert_status2.id)
+  #     expect(alert_status2.id).to_not be_nil
+  #   end
+  # end
```

Comment:
would prefer not to merge commented code, this can be added later in the followup PR after #13233